health.go

```diff
@@ -1105,9 +1105,9 @@ type TLSCert struct {
 type MinioHealthInfo struct {
 	Error string `json:"error,omitempty"`
 
-	Config      MinioConfig   `json:"config,omitempty"`
-	Info        MinioInfo     `json:"info,omitempty"`
-	Replication *ReplDiagInfo `json:"replication,omitempty"`
+	Config      MinioConfig     `json:"config,omitempty"`
+	Info        MinioInfo       `json:"info,omitempty"`
+	Replication *ReplDiagInfoV2 `json:"replication,omitempty"`
```

Comment:
Still making breaking changes, why are we doing that?

Reply:
The struct `ReplDiagInfo` got field types changed so created new `ReplDiagInfoV2`. I may need to introduce new type `MinioHealthInfoV2`, right?

Reply:
Added changes as per https://github.com/minio/madmin-go/pull/391#discussion_r2121125642